packages/playwright-test/src/webServer.ts

```diff
@@ -38,6 +38,24 @@ export class WebServer {
     this._isAvailable = getIsAvailableFunction(config);
   }
 
+  public static async createAll(configs: WebServerConfig[], reporter: Reporter): Promise<() => Promise<void>> {
+    const webServers: WebServer[] = [];
+    const killAll =  async () => {
+      for (const server of webServers)
+        await server.kill();
+    };
+
+    try {
+      for (const config of configs)
```

Comment:
NB: In the future, we could get fancy and start them all at once, but reporting and teardown are a bit ugglier for this optimization.